main.nf

```diff
@@ -939,6 +912,62 @@ process fetch_MTseqs {
 
 }
 
+process sentieon_mitochondrial_qc {
+
+    // Fetch mitochondrial coverage statistics
+    // Calculate mean_coverage and pct_above_500x
+    
+    cpus 52
+    memory '30 GB'
+	tag "$id"
+	cache 'deep'
+	time '2h'
+	scratch true
+	stageInMode 'copy'
+	stageOutMode 'copy'
+	container = "/fs1/resources/containers/sentieon_202112.sif"
+
+	when:
+	    params.antype == "wgs"
+    
+	input:
+        set group, id, file(bam), file(bai) from qc_mito_bam
+
+	output:
+    	set group, id, file("${id}_mito_coverage.tsv") into qc_mito
+		
+	"""
+	sentieon driver \\
+		-r $genome_file \\
+		-t ${task.cpus} \\
+		-i $bam \\
+		--algo CoverageMetrics \\
+        --cov_thresh 500 \\
+        mt_cov_metrics.txt
+
+    head -1 mt_cov_metrics.txt.sample_interval_summary > "${id}_mito_coverage.tsv"
+    grep "^M" mt_cov_metrics.txt.sample_interval_summary >> "${id}_mito_coverage.tsv"
+	"""
+}
+
+process build_mitochondrial_qc_json {
```

Comment:
The TSV file is converted to JSON in a separate process, to keep it open for any other future mito QC that might be merged from other channels into the qc.json file at this point.